src/components/Camera/measure_blur.js

```diff
@@ -0,0 +1,98 @@
+//ref: https://github.com/timotgl/inspector-bokeh/blob/master/measure_blur.js
```

Comment:
Fair enough.

What are your thoughts on moving it into a `lib/` directory or something, just to make clear that this is 3rd-party code that we're just pulling in?